src/components/DepositWidget/Row.tsx

```diff
@@ -89,7 +114,26 @@ export const Row: React.FC<RowProps> = ({ tokenBalances }: RowProps) => {
       </td>
       <td>{name}</td>
       <td title={formatAmountFull(exchangeBalanceTotal, decimals)}>{formatAmount(exchangeBalanceTotal, decimals)}</td>
-      <td title={formatAmountFull(withdrawingBalance, decimals)}>{formatAmount(withdrawingBalance, decimals)}</td>
+      <td title={formatAmountFull(withdrawingBalance, decimals)}>
+        {withdrawable ? (
+          <>
+            <button className="success" onClick={_withdraw} disabled={withdrawing}>
```

Comment:
For some reason, `withdrawing` state is not changing to `true` when `withdraw` is invoked, thus button is never disabled.